godot-macros/src/class/data_models/inherent_impl.rs

```diff
@@ -89,6 +93,19 @@ pub fn transform_inherent_impl(
     #[cfg(not(all(feature = "register-docs", since_api = "4.3")))]
     let docs = quote! {};
 
+    // This is the container struct that holds the names of all registered #[func]s.
+    // (The struct is declared by the macro derive_godot_class.)
```

Comment:
in most cases macros are being refereed by their user-side representation such as `#[derive(GodotClass)]`  or `#[godot_api]` instead of `macro derive_godot_class`. Should we keep this convention?

Reply:
also in other place it is being called "derive macro GodotClass" :sweat_smile: